### PR TITLE
AN136: Move default gasLimit to constants

### PR DESCRIPTION
**Pull request by vponline**
Move default gasLimit to constants, Jira: [AN136](https://api3dao.atlassian.net/jira/software/projects/AN/boards/18?selectedIssue=AN-136)

### CODE REVIEW

File: packages/airnode-node/src/evm/fulfillments/api-calls.test.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import * as fixtures from '../../../test/fixtures';
 import * as wallet from '../wallet';
 import { GasTarget, RequestErrorMessage, RequestStatus } from '../../types';
 import { AirnodeRrp } from '../contracts';
-import { MAXIMUM_ONCHAIN_ERROR_LENGTH } from '../../constants';
+import { MAXIMUM_ONCHAIN_ERROR_LENGTH, GAS_LIMIT } from '../../constants';
```
Comment (Siegrift) [inline packages/airnode-node/src/evm/fulfillments/api-calls.test.ts:22]: Why importing it in this test and not the others (e.g. process-transactions.test)

Comment (vponline) [reply packages/airnode-node/src/evm/fulfillments/api-calls.test.ts:22]: That was fast! I was actually going to ask if we should use the same constant in every place with the same `gasLimit` value, or only in the places where it had been defined as a constant previously.

Comment (Siegrift) [reply packages/airnode-node/src/evm/fulfillments/api-calls.test.ts:22]: Feel free to use it as variable everywhere.

File: packages/airnode-node/src/constants.ts [hunk 2]
```diff
@@ -50,3 +50,6 @@ export const BASE_FEE_MULTIPLIER = 2;
 
 // The maximum character length of an error message sent on-chain
 export const MAXIMUM_ONCHAIN_ERROR_LENGTH = 100;
+
+// The default gas limit
+export const GAS_LIMIT = 500_000;
```
Comment (andreogle) [inline packages/airnode-node/src/constants.ts:55]: Nit: It's not clear what this "gas limit" applies to. I would prefer a more descriptive name here if possible. 

```suggestion
export const DEFAULT_FULFILLMENT_GAS_LIMIT = 500_000;
```
or maybe `API_CALL_FULFILLMENT_GAS_LIMIT` (?)

Also, the comment above doesn't really add much in it's current state

**Review by andreogle — APPROVED** [review]

File: packages/airnode-node/src/constants.ts [hunk 3]
```diff
@@ -50,3 +50,6 @@ export const BASE_FEE_MULTIPLIER = 2;
 
 // The maximum character length of an error message sent on-chain
 export const MAXIMUM_ONCHAIN_ERROR_LENGTH = 100;
+
+// The default gas limit for API fulfillment transactions
+export const API_CALL_FULFILLMENT_GAS_LIMIT = 500_000;
```
Comment (andreogle) [inline packages/airnode-node/src/constants.ts:55]: Nice 👍🏻

**Review by andreogle — APPROVED** [review]

**Pull request merged**